examplestreams/src/main/java/com/felhr/examplestreams/UsbService.java

```diff
@@ -211,7 +211,7 @@ private void setFilter() {
      */
     private void requestUserPermission() {
         Log.d(TAG, String.format("requestUserPermission(%X:%X)", device.getVendorId(), device.getProductId() ) );
-        PendingIntent mPendingIntent = PendingIntent.getBroadcast(this, 0, new Intent(ACTION_USB_PERMISSION), 0);
+        PendingIntent mPendingIntent = PendingIntent.getBroadcast(this, 0, new Intent(ACTION_USB_PERMISSION), PendingIntent.FLAG_IMMUTABLE);
```

Comment:
I got no ideas the details here.  But it seems 0 would be signifying on flags, and PendingIntent.FLAG_IMMUTABLE is 0x04000000.

So I assume you found that for some reason that flag is necessary?

Reply:
This change is required if you target Android 12. 
It's on the example codebase, won't have any implication on the library itself.

https://developer.android.com/about/versions/12/behavior-changes-12#pending-intent-mutability